WordPress/Classes/ViewRelated/Stats/Subscribers/SubscribersChartMarker.swift

```diff
@@ -0,0 +1,35 @@
+import DGCharts
+
+class SubscribersChartMarker: StatsChartMarker {
+    let date: Date
+
+    init(dotColor: UIColor, name: String, date: Date) {
+        self.date = date
+        super.init(dotColor: dotColor, name: name)
+    }
+
+    public required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func text(for entry: ChartDataEntry) -> NSMutableAttributedString? {
+
+        let subscriberCountRowAttributes: [NSAttributedString.Key: Any] = [.font: UIFont.preferredFont(forTextStyle: .headline),
+                                                                           .paragraphStyle: paragraphStyle,
+                                                                           .foregroundColor: UIColor.white]
+
+        let subscriberLabelRowAttributes: [NSAttributedString.Key: Any] = [.font: UIFont.preferredFont(forTextStyle: .footnote),
+                                                                           .paragraphStyle: paragraphStyle,
+                                                                           .foregroundColor: UIColor.white]
+
+        let subscriberDateRowAttributes: [NSAttributedString.Key: Any] = [.font: UIFont.preferredFont(forTextStyle: .footnote),
+                                                                          .paragraphStyle: paragraphStyle,
+                                                                          .foregroundColor: UIColor.white.withAlphaComponent(0.8)]
+
+        let attrString = NSMutableAttributedString()
+        attrString.append(NSAttributedString(string: entry.y.abbreviatedString() + "\n", attributes: subscriberCountRowAttributes))
+        attrString.append(NSAttributedString(string: "\(name)\n", attributes: subscriberLabelRowAttributes))
+        attrString.append(NSAttributedString(string: DateValueFormatter().dateFormatter.string(from: date), attributes: subscriberDateRowAttributes))
+        return attrString
```

Comment:
The issue with this approach is that "8 Views" isn't reversed for languages where the number follows the noun.

Reply:
Good observation.

When it's written vertically I wonder if it should be read like a sentence or more like a title and subtitle. Maybe then the language order doesn't matter as much?

```
8
Subscribers
Apr 11
```

Reply:
I'm not sure. I'm ok with leaving it out-of-scope of this PR because it's not a straightforward fix and as you say, it might not be such a big issue after all. I'll create a separate issue to cover both the Subscriber chart markers and the Views & Visitor chart markers, so we can decide there what to do.